engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/TypeDefinitionSet.scala

```diff
@@ -0,0 +1,49 @@
+package pl.touk.nussknacker.engine
+
+import cats.data.{NonEmptyList, Validated}
+import cats.data.Validated.{Invalid, Valid}
+import org.apache.commons.lang3.ClassUtils
+import org.springframework.expression.EvaluationContext
+import org.springframework.expression.spel.ExpressionState
+import org.springframework.expression.spel.ast.TypeReference
+import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.expression.ExpressionParseError
+import pl.touk.nussknacker.engine.api.typed.typing.{TypedClass, TypingResult}
+import pl.touk.nussknacker.engine.definition.{DefinitionExtractor, ProcessDefinitionExtractor, TypeInfos}
+import pl.touk.nussknacker.engine.spel.TypedNode
+import pl.touk.nussknacker.engine.spel.ast.SpelAst.RichSpelNode
+
+
+object TypeDefinitionSet {
+
+  def empty: TypeDefinitionSet = TypeDefinitionSet(Set.empty)
+
+  def withDefaultClasses: TypeDefinitionSet = {
```

Comment:
I don't think this should be here - in production code. Let's move it to tests..

Reply:
Moved to test, thanks